docker-compose.debug.yml

```diff
@@ -1,42 +1,44 @@
 version: '3.8'
 services:
   webapp:
-    build: .
-    command: "npm run build:run"
+    build:
+      context: .
+      target: dev
```

Comment:
With multistage dockerfiles, you can't just say build . Bc that builds every image. You'll need a target (reminds me of Makefiles actually).

This also removes the need for overriding the command; you specify one CMD for each image in the multistage dockerfile.